### PR TITLE
Ordered dump

### PR DESCRIPTION
**Pull request by lparam**
* Ordered dump
* Insert blank line between section

### CODE REVIEW

**Comment by amjith** [comment]
I would love to see this PR merged and released. Is there anything I can help to make that happen? 

Perhaps testing or adding docs?

File: toml.py [hunk 1]
```diff
@@ -793,15 +794,16 @@ def dumps(o, preserve=False):
...
+    return retval[:-1]
 
 
-def _dump_sections(o, sup, preserve=False):
+def _dump_sections(o, sup, preserve=False, _dict=dict):
     retstr = ""
     if sup != "" and sup[-1] != ".":
         sup += '.'
-    retdict = {}
+    retdict = _dict()
```
Comment (uiri) [inline toml.py]: Wouldn't it make more sense to just use `o.__class__()` here?

Comment (amjith) [reply toml.py]: Yes. That will solve the use case. I'll open a PR with that change.

Comment (Greg-Szabo) [reply toml.py]: Hi @amjith , is this change moving along? Do you need any help with it?

Comment (amjith) [reply toml.py]: I've already made a PR #121 that solves this issue. It has been merged. 

This PR can be closed.

**Comment by uiri** [comment]
Does #121 address this? Can this PR be closed?

**Comment by uiri** [comment]
Hi,

I am closing this since I believe this was fixed by #121. If you believe this to be a mistake, feel free to re-open the pull request. Thank you for taking the time to create the patch.

**Pull request closed without merging**